modules/request-quota-increase/variables.tf

```diff
@@ -1,10 +1,10 @@
 variable "resources_to_increase" {
-  description = "A map where the key is the resource and the value is the desired quota. The only resources supported at the moment are 'nacl_rules' and 'nat_gateway'. You can also use the `aws_servicequotas_service_quota` resource directly, there are instructions on how to find the Service Code and Quota Code on the README!"
+  description = "A map where the key is the resource and the value is the desired quota. The only services supported at the moment are VPC and IAM. You can also use the `aws_servicequotas_service_quota` resource directly!"
```

Comment:
We should tell users where to look up the available values. 

Also, what does "You can also use the aws_servicequotas_service_quota resource directly" mean?

Reply:
> Also, what does "You can also use the aws_servicequotas_service_quota resource directly" mean?
That if there is not available not the module, they can use just the resource.

Reply:
Updated! 1e19c1e